src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -1639,7 +1639,7 @@ public Report getSubReport(String subReportId, Set<String> severityLevels) {
 
     @Transactional(readOnly = true)
     public List<Report> getNodeReport(UUID nodeUuid, String reportId, ReportType reportType, Set<String> severityLevels) {
-        return reportService.getReport(UUID.fromString(reportId), nodeUuid.toString(), nodeUuid + "@" + reportType.reportKey, ReportNameMatchingType.EXACT_MATCHING, severityLevels);
```

Comment:
reportType is not used anymore: can be remove from here, rest api, and front-end